modules/oxide/gulpfile.js

```diff
@@ -154,10 +154,16 @@ gulp.task('copyToEnd', function () {
     .pipe(gulp.dest('../../../end/src/assets/tinymce'));
 })
 
+gulp.task('copyDarkModeToEnd', function () {
```

Comment:
I haven't worked with gulp in a long time, but couldn't this task be added to `copyToEnd`? E.g., by putting the dark skin in a sub-folder of the `wagtail` skin? :thinking: